Release on F-Droid

I can help with this if needed.

@cgutman I just gave your build recipe a try on F-Droid buildserver and I'm afraid it failed.

error in `fdroid build --server com.limelight` output:
```
configure: error: in `/home/vagrant/build/srclib/opus':
configure: error: C compiler cannot create executables
See `config.log' for more details
```
(full log: http://paste.c-net.org/MackeyFountain)

```
./configure: line 3600: armv7a-linux-androideabi16-clang: command not found
configure:3602: $? = 127
configure:3640: result: no
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "opus"
| #define PACKAGE_TARNAME "opus"
| #define PACKAGE_VERSION "1.3"
| #define PACKAGE_STRING "opus 1.3"
| #define PACKAGE_BUGREPORT "opus@xiph.org"
| #define PACKAGE_URL ""
| /* end confdefs.h.  */
| 
| int
| main ()
| {
| 
|   ;
|   return 0;
| }
configure:3645: error: in `/home/vagrant/build/srclib/opus':
configure:3647: error: C compiler cannot create executables
See `config.log' for more details
```

full output of: _build/srclib/opus/config.log_: https://paste.c-net.org/ProdigalOpera

Any ideas what the problem might be?

--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@ Check our [wiki](https://github.com/moonlight-stream/moonlight-docs/wiki) for mo
 ## Installation
 
 * Download and install Moonlight for Android from
-[Google Play](https://play.google.com/store/apps/details?id=com.limelight), [Amazon App Store](http://www.amazon.com/gp/product/B00JK4MFN2), or directly from the [releases page](https://github.com/moonlight-stream/moonlight-android/releases)
+[Google Play](https://play.google.com/store/apps/details?id=com.limelight), [F-Droid](https://f-droid.org/packages/com.limelight/), [Amazon App Store](http://www.amazon.com/gp/product/B00JK4MFN2), or directly from the [releases page](https://github.com/moonlight-stream/moonlight-android/releases)
 * Download [GeForce Experience](http://www.geforce.com/geforce-experience) and install on your Windows PC
 
 ## Requirements

